src/wallet/generateFaucetWallet.ts

```diff
@@ -292,7 +304,7 @@ function getFaucetUrl(client: Client): FaucetNetwork | undefined {
     return FaucetNetwork.Devnet
   }
 
-  return undefined
+  throw new XRPLFaucetError('Faucet URL is not defined or inferrable.')
```

Comment:
Thoughts on adding a special error message for standalone if they're trying to connect to localhost?

Reply:
I don't see why we would, I think this message covers that case.